src/Mantasflowers.WebApi/Setup/DI/AuthenticationModule.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Autofac;
+using Mantasflowers.Persistence.Authentication;
+
+namespace Mantasflowers.WebApi.Setup.DI
+{
+    public class AuthenticationModule : Module
+    {
+        protected override void Load(ContainerBuilder builder)
+        {
+            builder.RegisterType<FirebaseContext>()
+                .SingleInstance();
```

Comment:
Maybe this should be `InstancePerRequest`? Just thinking if having a single firebase instance wouldn't be blocking other calls.